Ghidra/Features/Base/src/main/java/ghidra/app/script/ScriptInfo.java

```diff
@@ -197,9 +195,15 @@ private void parseHeader() {
 					break;
 				}
 
-				if (DOCUMENTATION_START.matcher(line).find()) {
-					while (line != null && !DOCUMENTATION_END.matcher(line).find()) {
+				Pattern blockStart = provider.getBlockCommentStart();
```

Comment:
if `GhidraScriptProvider#getBlockCommentStart` returns a `null`, you can test for it here and avoid the loop.